### DISCUSSION TITLE
Rename Error

### DISCUSSION THREAD

**Discussion by steigenn** [hashlips-lab/nft-erc721-collection#429] [Q&A] [answered]
It gives the following error in `yarn rename-contract NFTNAME` command. What do I need to do to solve it?

`Internal Error: @hashlips-lab/nft-erc721-collection-smart-contract@workspace:.: This package doesn't seem to be present in your lockfile; run "yarn install" to update the lockfile at pQ.getCandidates (C:\Users\User\AppData\Local\node\corepack\yarn\3.3.0\yarn.js:435:5146) at Wf.getCandidates (C:\Users\User\AppData\Local\node\corepack\yarn\3.3.0\yarn.js:391:1264) at C:\Users\User\AppData\Local\node\corepack\yarn\3.3.0\yarn.js:439:7695 at Rf (C:\Users\User\AppData\Local\node\corepack\yarn\3.3.0\yarn.js:390:8965) at ge (C:\Users\User\AppData\Local\node\corepack\yarn\3.3.0\yarn.js:439:7675) at async Promise.allSettled (index 0) at async eo (C:\Users\User\AppData\Local\node\corepack\yarn\3.3.0\yarn.js:390:8293) at async C:\Users\User\AppData\Local\node\corepack\yarn\3.3.0\yarn.js:439:8205 at async ni.startProgressPromise (C:\Users\User\AppData\Local\node\corepack\yarn\3.3.0\yarn.js:390:45507) at async He.resolveEverything (C:\Users\User\AppData\Local\node\corepack\yarn\3.3.0\yarn.js:439:6238) `

**Comment by liarco** [comment]
Hi @steigenn,
Quick tips that I hope will help you with this:
1) are you running that command from inside the `smart-contract` folder? If you open your terminal and run the command straight from the project root folder it won't work
2) did you install the dependencies using `yarn` alone (inside the contract folder) before running your command? The contract and dapp folders are two separated projects, so dependencies must be installed from inside those 2 folders independently

Our time for dedicated support (not bugs) is very limited so if you need more help you may consider joining the [Open Devs Crew](https://opendevs.io), we have a dedicated channel on the HashLips discord server where you can link you questions from GitHub and we will give them a higher priority.

Let me know if you solved the problem!

**Reply by steigenn** [reply]
Hello, @liarco 

1. How to create NFT collection on Hashlips youtube channel - I am practicing the steps in the Masterclass video. 

2. In the video, the installation is done with the `yarn` command.

There has been no response on the Discord server for a while.

**Reply by liarco** [reply]
@steigenn, just a quick info about the discord server: there are a lot of users that may be able/willing to help for free there, but it really depends on availability so I understand what you said about getting no response. The Open Devs Crew that I mentioned is a dedicated group of people (including me and other devs who maintain these tools) who help each other in a dedicated channel.

You need an NFT from our collection to be able to join the channel on discord, but once you join there you can get in touch with us directly.

**Comment by steigenn** [comment, marked as answer]
I solved the problem. "https://www.youtube.com/watch?v=zjlg-0622PU" I followed the steps in this video and my problem was solved.

**Reply by liarco** [reply]
@steigenn Great! Thank you for taking time to update the discussion. Good luck with your project!
